ouroboros-consensus/demo-playground/Run.hs

```diff
@@ -174,24 +224,37 @@ handleSimpleNode p CLI{..} myNodeAddress (TopologyInfo myNodeId topologyFile) =
       myAddr:_ <- case myNodeAddress of
         NodeAddress host port -> getAddrInfo Nothing (Just host) (Just port)
 
+      myLocalAddr <- mkLocalSocketAddrInfo myNodeId
+
       -- TODO: cheap subscription managment, a proper one is on the way.  The
       -- point is that it only requires 'NetworkApplications' which is a thin
       -- layer around 'MuxApplication'.
 
+      -- serve local clients (including tx submission)
+      localServer <-
+        forkLinked registry $
+          NodeToClient.withServer
```

Comment:
It's important to note that this version of `withServer` is not doing error handing, and here the DNS management will not fix it as it does not applies.  We should modify the version exposed in `NodeToClient` to handle exceptions properly. We'd need to think what kind of exception handling behaviour we want from a local client / node communication.

Reply:
The behaviour for local clients should be basically the same policy one uses in a http server: exceptions in talking to a client are logged but otherwise discarded. There's no reputation scoring like with remote peers.